### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by digomes87**
pull request for validation with tse

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+
+//The book class
```
Comment (azdrenymeri) [inline js/main.js]: Remove all comments starting with this one

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+class Book {
+  constructor(title, author, pages, status, id) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+    this.pages = pages;
+    this.status = status;
+  }
+
+  togglestat() {
```
Comment (azdrenymeri) [inline js/main.js]: change the name of the function to a `camelCase` one or just name it `toggle`

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+			<td>${b.title}</td>
+			<td>${b.pages}</td>
+			<td><input type="button" onclick="statchange(${b.id})"
+				value="${b.status}"></td>
+			<td><input type="button" onclick="dell(${b.id})"
+				value="delete"> </td>
+			`;
+    tb.append(tr);
+  }
+}
```
Comment (azdrenymeri) [inline js/main.js]: I'm guessing this is another implementation of  `render()` function?
scr

File: js/main.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+  libone.addtolib();
+}
+
+function dell(rwid) {
+  libone.deleBook(rwid);
+}
+
+function statchange(id) {
+  libone.stati(id);
+}
```
Comment (azdrenymeri) [inline js/main.js]: These two functions are just declared but never used so just remove them.

File: js/main.js [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+				value="${b.status}"></td>
+			<td><input type="button" onclick="dell(${b.id})"
+				value="delete"> </td>
+			`;
+    tb.append(tr);
+  }
+}
+
+var lib = [];
+var iid = 0;
```
Comment (azdrenymeri) [inline js/main.js]: Use the ES6 `const` and `let`

**Review by azdrenymeri — CHANGES_REQUESTED** [review]
#### Hello @digomes87 and  @onedebos 👋 
* The project is marked as collaborative, why only  @digomes87 committed/pushed code?
* It would be nice to have validation on the book from:
(I was able to create empty books)
scr
#### I have suggested some changes on your `main.js` code. Kindly implement those and add the project for another review.

File: js/main.js (hunk 4 above)
Comment (digomes87) [reply js/main.js]: Okey, but if I remove my function dell, I'll can't delete my list more.

File: js/main.js (hunk 1 above)
Comment (digomes87) [reply js/main.js]: Okey, commets deleted

File: js/main.js (hunk 2 above)
Comment (digomes87) [reply js/main.js]: Okey, updated

File: js/main.js (hunk 3 above)
Comment (digomes87) [reply js/main.js]: fix that.

File: js/main.js (hunk 5 above)
Comment (digomes87) [reply js/main.js]: updated

**Review by Oghenebrume50 — APPROVED** [review]
STATUS: APPROVED

Looks good, *This is approved*

RaphaelNoriode :sunglasses:

**Pull request merged**